apps/machine/zynqmp_r5/platform_info.c

```diff
@@ -45,20 +45,21 @@
 #define SHARED_MEM_SIZE 0x100000UL
 #define SHARED_BUF_OFFSET 0x8000UL
 
-
+#ifndef RPMSG_NO_IPI
```

Comment:
what is the nominal use case? with IPI or without IPI.
I wonder if the switch should be RPMSG_IPI, that would make code more readable....

Reply:
the nominal use case is with IPI. so I can change to instead have just switch based on if no ipi switch is set if that is more proper.